mkdocs.yml

```diff
@@ -26,16 +26,16 @@ pages:
     - "Create your first app": dev/app.md
     - "Develop a connector": dev/konnector.md
     - "Create a mobile app with cordova": dev/cordova.md
-theme: readthedocs
-theme_dir: 'cozy-theme'
+theme: mkdocs
+# theme_dir: 'cozy-theme'
 markdown_extensions:
   - admonition
   - codehilite
   - extra
   - footnotes
-  - markdown_i18n:
-      i18n_lang: en_US
-      i18n_dir: i18n
+#  - markdown_i18n:
```

Comment:
Please don't commit this changes, they just allow to run mkdocs on your computer without the markdown_i18n plugin.